projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/review-submit/review-submit.component.spec.ts

```diff
@@ -106,7 +107,7 @@ describe('ReviewSubmitComponent', () => {
         MockCartItemListComponent,
         MockCardComponent
       ],
-      providers: [CheckoutService, CartDataService]
+      providers: [CheckoutService, CartDataService, UserService]
```

Comment:
As this is a unit test, maybe it's better to inject a mocked/stubbed `UserService` instead of the actual instance?